### PR TITLE
Fix paths and typos for macOS scripts

### LINKED ISSUE #4737: [Bug]Can't build on macOS after recent update.
### Description of the issue

"MSBuild version 17.5.1+f6fdcf537 for .NET
MSBUILD : error MSB1009: Project file does not exist.
Switch: Ryujinx.Ava"

I get this msb1009 when trying to build with the latest updates. it worked yesterday before it was updated. 

### Reproduction steps

just built the normal way. cloned the GitHub and cd'd to the cloned folder. typed "./distribution/macos/create_macos_release.sh ./ ./distribution/macos/temp ./distribution/macos/output ./distribution/macos/entitlements.xml 1.1.1 0" then got the error above.

### Log file

n/a

### OS

Mac OS ventura 13.3.1

### Ryujinx version

newest 

### Game version

_No response_

### CPU

_No response_

### GPU

_No response_

### RAM

_No response_

### List of applied mods

_No response_

### Additional context?

_No response_

### PR DESCRIPTION
**Pull request by TSRBerry**
This was missed in #4725.

---

Fixes #4737

### CODE REVIEW

**Review by gdkchan — APPROVED** [review]
Code lgtm, I assume that you tested that the build script is working locally. Thanks for the quick fix.

**Comment by TSRBerry** [comment]
I haven't yet, I'll be able to test locally tomorrow.

I definitely should have added that to the description, sorry!

**Comment by TimeZlicer** [comment]
Good day

I tested locally and it builds successfully, binary runs fine (able to launch a game)

```
git clean -fdx; DOTNET_CLI_TELEMETRY_OPTOUT=1 sh -x ./distribution/macos/create_macos_release.sh . ./temp ./output distribution/macos/entitlements.xml `git describe --tags`-macos `git describe --always`
```

File: distribution/macos/create_app_bundle.sh [hunk 1]
```diff
@@ -41,12 +41,11 @@ then
         exit 1
     fi
 
-    # NOTE: Currently require https://github.com/indygreg/apple-platform-rs/pull/44 to work on other OSes.
+    # NOTE: Currently requires https://github.com/indygreg/apple-platform-rs/pull/44 to work on other OSes.
     # cargo install --git "https://github.com/marysaka/apple-platform-rs" --branch "fix/adhoc-app-bundle" apple-codesign --bin "rcodesign"
```
Comment (marysaka) [inline distribution/macos/create_app_bundle.sh:45]:
```suggestion
    # cargo install apple-codesign
```

File: distribution/macos/create_app_bundle.sh [hunk 2]
```diff
@@ -41,12 +41,11 @@ then
         exit 1
     fi
 
-    # NOTE: Currently require https://github.com/indygreg/apple-platform-rs/pull/44 to work on other OSes.
+    # NOTE: Currently requires https://github.com/indygreg/apple-platform-rs/pull/44 to work on other OSes.
```
Comment (marysaka) [inline distribution/macos/create_app_bundle.sh:44]:
```suggestion
```

**Review by marysaka — APPROVED** [review]

**Pull request merged**